packages/numpy/meta.yaml

```diff
@@ -1,16 +1,24 @@
 package:
   name: numpy
-  version: 1.25.2
+  version: 1.26.1
   tag:
     - min-scipy-stack
   top-level:
     - numpy
 source:
-  url: https://files.pythonhosted.org/packages/a0/41/8f53eff8e969dd8576ddfb45e7ed315407d27c7518ae49418be8ed532b07/numpy-1.25.2.tar.gz
-  sha256: fd608e19c8d7c55021dffd43bfe5492fab8cc105cc8986f813f8c3c048b38760
+  url: https://files.pythonhosted.org/packages/78/23/f78fd8311e0f710fe1d065d50b92ce0057fe877b8ed7fd41b28ad6865bfc/numpy-1.26.1.tar.gz
+  sha256: c8c6c72d4a9f831f328efb1312642a1cafafaa88981d9ab76368d50d07d93cbe
 
 build:
-  backend-flags: --disable-optimization
+  # numpy uses vendored meson, so we need to pass the cross file manually
+  backend-flags: |
+    setup-args=-Dallow-noblas=true
+    setup-args=--cross-file=${MESON_CROSS_FILE}
+  # numpy creates numpy/distutils/__pycache__ directory during the build.
+  # It breaks our test because there is a .pyc in the directory.
```

Comment:
This is caused by a direct import of a utility with:
```python
    spec = importlib.util.spec_from_file_location('conv_template', genapi_py)
    mod = importlib.util.module_from_spec(spec)
    spec.loader.exec_module(mod)
```

I can rework that if it's a problem, and add a test to prevent a regression. What is the constraint here, no `.pyc` files under `numpy/`? Is it okay for there to be some under `tools/` instead (so outside the package source tree)?

Reply:
I don't think this is a big enough deal that you should think about changing numpy. It's pretty normal for there to be build time imports, we just have to deal with it. We could just always delete `__pycache__` folders for all packages.

Reply:
Right, it isn't a big issue.